server/src/ts/games/cards/decks.ts

```diff
@@ -43,29 +43,35 @@ export abstract class Deck<C extends Card.BaseCard> {
     this.discarded.add(card);
   }
 
-  public draw(cards: 1): [C];
-  public draw(cards: 2): [C, C];
-  public draw(cards: 3): [C, C, C];
-  public draw(cards: number): C[];
-  public draw(cards: number): C[] {
+  public draw(cards: 1, bot?: boolean): [C];
+  public draw(cards: 2, bot?: boolean): [C, C];
+  public draw(cards: 3, bot?: boolean): [C, C, C];
+  public draw(cards: number, bot?: boolean): C[];
+  public draw(cards: number, bot?: boolean): C[] {
     const cardsLeft = this.cards.length;
     const toDraw = Math.min(cardsLeft, cards);
-    const result = this.cards.splice(0, toDraw);
+    const result = this.cards
+      .splice(0, toDraw)
+      .filter((card) => card.source.source !== "Custom");
```

Comment:
This may mean that blank cards aren't being discarded properly when picked up by the bot. Not sure?